docs/release-notes/mia-platform-v11.mdx

```diff
@@ -0,0 +1,114 @@
+
+Don't miss the opportunity to explore all the incredible new features of Mia-Platform V11 and discover how we can help you transform your digital enterprise, providing an optimized development experience centered on developers' needs.
+Mia-Platform V11, Simplify Complexity.
+
+## Console
+
+### Providers Management
+
+We have developed the Mia-Platform Console so that it can integrates properly with the best and most widely used tools and providers on the market. 
+
+However, this integration needs a proper governance system, and this feature has just that goal: to make it easier to keep constantly documented and ordered all the Providers you want the console to use. Through the new dedicated section - named "Providers" - inside the "Company Overview" area it will be possible to manage the creation, configuration, and editing of the providers used by Mia-Platform Console (GitProvider, Secret Manager and CI/CD Tools).
+
+![Default Providers Selection](./img/v10.9_defaultProvider.png)
+
+Furthermore, again through the same section, it will be possible to set the default Providers that will be used in all new projects created in the respective company. This will enhance the governance of Company Owners on all the Projects of their Companies, granting them better control over the management of company tools.
+
+### Machine-to-Machine Authentication
+
+Service accounts are an essential component of many IT environments, allowing applications and services to access various resources and perform operations without the need for human intervention.
+Mia-Platform Console allows to create and manage service accounts, useful for automating processes that previously had to be done by a human user.
+Assign roles and permissions to service accounts lets you to improve speed, reliability and scalability of your CI/CD workflows.
+For instance, service accounts are useful for applications that require continuous monitoring, reporting, and data processing, or you can give the permission to trigger automatic deploys, which can significantly streamline the process of deploying code changes to an application.
+Service accounts are also potentially great for security. Using service accounts promotes the separation of roles and responsibilities, which reduces the risk of credential misuse or theft.
+
+### Webhooks Management
+
+Each user has different needs, especially dependent on their business. We believe it is critical to give each user the ability to manage any additional custom flows as certain events occur in the console.
+
+For this reason, we have introduced the ability for the user to be able to subscribe to specific events in Console via HTTP webhooks via our CMS, so that custom HTTP calls can be configured that will be triggered when the respective event occurs in Console.
+
+For now, the supported events are:
+
+-Project Creation
+-Service Creation
```

Comment:
```suggestion
Each user has different needs, especially dependent on their business. We believe it is critical to give each user the ability to manage any additional custom flows as certain events occur in the Console.

For this reason, we have introduced the possibility for the user to subscribe to specific events in Console via HTTP webhooks, through our CMS. With this feature it will be possible to configure custom HTTP calls that will be triggered when the respective event takes place in Console

For now, the supported events are:

*Project Creation  
*Service Creation
```